rtris.py

```diff
@@ -4,11 +4,176 @@
 from numpy import add as np_add
 os.environ['PYGAME_HIDE_SUPPORT_PROMPT'] = "hide"
 import pygame,pygame.freetype
+from sys import argv
 
 K_DROP=False
 confpath=os.path.abspath(os.path.expanduser("~/.rtrisconf"))
 
+VERSION="1.0a3"
+COPYRIGHT_YEAR="2019"
+# TODO: do you wanna give out your full name?
+COPYRIGHT_HOLDER="RIEDLER"
+AUTHORS="RIEDLER and Michael Federczuk"
+
+USAGE="usage: %s" % (argv[0])
```

Comment:
I don't really get this... argv[0] is just the path to the python file.

Reply:
Usages always print the executable name at the beginning, it just looks better.  
We could change it to be something like this: `"usage: python3 %s" % (argv[0])`, if you want that

Reply:
I've never seen that, but I don't really see why it shouldn't be there. Nah I think your originial version is good.